keyboards/themadnoodle/noodlepad_micro/readme.md

```diff
@@ -0,0 +1,30 @@
+# NoodlePad [Micro]
+
+![NoodPad[Mircro]](https://imgur.com/a/l1iZjWX)
+
+The Noodlepad [Micro] is a 6 key 2 encoder macro keypad designed using RP2040 chipset
```

Comment:
```suggestion
The NoodlePad [Micro] is a 6 key 2 encoder macro keypad designed using RP2040 chipset.
```

Reply:
Hahaha I really need to work on my spell checking 😅